deploy/chart/values-edge.yaml

```diff
@@ -1,11 +1,33 @@
 deploy:
   ecs: false
 
-csi:
-  disk:
-    enabled: false
-  local:
-    enabled: true
-  oss:
-    controller:
-      enabled: false
+nodepools:
+  default:
+    pluginname: csi-plugin
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: type
+          operator: NotIn
+          values:
+            - virtual-kubelet
+        - key: alibabacloud.com/is-edge-worker
+          operator: In
+          values:
+            - "false"
+  edge:
+    pluginname: csi-plugin-edge
+    csi:
+      disk:
+        enabled: false
+      local:
+        enabled: true
```

Comment:
We should move this to the top level `csi`, because controller only looking for that, and we need local plugin controller.

Reply:
why original local.controller equals to true, it seems to make no sense. how about set it to false? 
`{{- if and .Values.csi.local.enabled .Values.csi.local.controller.enabled }}`

Reply:
Because `.csi.local.controller.enabled` is introduced later. And we should maintain capability to old configs, both for users who enabled or disabled local driver.

Reply:
Now you moved it to top level, remove this one?

Reply:
ok